front/src/common-app/components/vote-options.component.tsx

```diff
@@ -33,13 +35,13 @@ export const VoteOptionsComponent: React.FC<Props> = props => {
     voteCollection.length === 1 ? classes.contanierLabelShowVote : '';
 
   return (
-    <div className={classes.container}>
+    <div className={classes.container} ref={elementFocus} tabIndex={0}>
```

Comment:
Move ref and tabIndex to `Typography h2 Your Vote...` component

Reply:
Fix